internal/cli/cmd/get/get.go

```diff
@@ -57,12 +57,10 @@ func (o *option) Complete(args []string) error {
 		return err
 	}
 
-	// Complete the output only if it has been set by the user
-	if len(o.Output) > 0 {
```

Comment:
you shouldn't do that as the default format for the `get` command is a table (that's the only excepting existing in the CLI about the format supported)